### PR TITLE
Maintenance/update logo for doc

### PR DESCRIPTION
**Pull request by kalebphipps**
# Description

Update the ``ARTIST`` logo for the docs and access this logo for the Readme to fix problems with the logo not being presented properly. 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kalebphipps** [comment]
Extra logos have now been removed and all logos are in the ``logos`` folder in the ``root``. This fixes Issue #79.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-<img src="logo.svg" alt="logo" width="500"/>
+<img src="logos/artist_logo.png" alt="logo" width="500"/>
```
Comment (Markus-Goetz) [inline README.md:2]: Should we not use the SVG by default?

File: docs/conf.py [hunk 2]
```diff
@@ -39,7 +41,7 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static/"]
 html_css_files = ["custom.css"]
-html_logo = "images/logo_transparent.png"
+html_logo = f"{ARTIST_ROOT}/logos/artist_logo.png"
```
Comment (Markus-Goetz) [inline docs/conf.py:44]: Should we not use the SVG by default?

**Review by Markus-Goetz — COMMENTED** [review]
Should we not link to the SVG

File: README.md (hunk 1 above)
Comment (kalebphipps) [reply README.md]: Yep - Sphinx was causing issues before but this now seems to have been resolved. I have switched both logos to SVG!

**Review by Markus-Goetz — APPROVED** [review]

**Pull request merged**